bin/openosp-setup.sh

```diff
@@ -78,11 +81,17 @@ if [ ! -f ./volumes/oscar.properties ]; then
   if [ $LOCATION == 'ontario' ]
   then
     echo "Using Ontario properties"
-    cp docker/oscar/conf/oscar_mcmaster_on.properties ./volumes/oscar.properties
-
+    cp docker/oscar/conf/oscar_mcmaster_phc.properties ./volumes/oscar.properties
+    echo "logintitle=${CLINIC_NAME}" >> ./volumes/oscar.properties
+    echo "logintext=${CLINIC_TEXT} ${CLINIC_LINK}" >> ./volumes/oscar.properties
+    echo "CLINIC_LINK=${CLINIC_LINK}" >> ./volumes/oscar.properties
+    THE_DATE=$(date +"%Y.%m.%d")
```

Comment:
Should not need to copy the build date and time from the oscar properties file.  Oscar should display these on it's own.